chapters/Chapter 1 — Untitled.md

```diff
@@ -87,38 +86,46 @@ Leanne took an explosive from her belt, and planted it on the hull, just behind
 
-Leanne’s other attacker flew out of the hole. A bullet followed her. Leanne supposed it hadn’t been necessary: the other attacker wasn’t wearing a suit. She’d be out in thirty seconds, and dead in another thirty, unless her Alt could pull off a miracle. But Leanne’s rule was Leanne’s rule.
+Leanne’s other attacker flew out of the hole. A bullet followed them. Leanne supposed it hadn’t been necessary: the other attacker wasn’t wearing a suit. She’d be unconscious in thirty seconds, and dead in another thirty, unless her Alt could pull off a miracle. But Leanne couldn’t take chances. She had to see it.
 
 Leanne forced herself to watch until the end. Her aim hadn’t been perfect. She’d hit the woman’s neck instead of her head.
 
 The woman’s twitching slowed. The fury in her boiling eyes dimmed. 
 
-Leanne took a deep breath, and crawled into her attacker’s ship. Her head spun as she landed in the remains of the cockpit. 
+Leanne took a deep breath, and crawled into her attacker’s ship. Her vision swum as she landed in the remains of the cockpit. 
+
+She checked the feeds from the cameras. It appeared she was alone, or alone enough. She rifled through the console. Her attackers, like her, were assassins. They should have a card for the job. It would have all the pertinent details. They’d turn it in with proof of death. 
+
+It was unlikely there’d be anything on the card to trace back to the submitter; assassins always worked through an arranger, and even the arranger rarely knew who hired them. But Leanne searched all the same, whether out of habit or hunch.
+
+The cards was small. There was a little yellow insignia on the upper-left of a diving eagle: the Leesomes, if Leanne remembered correctly. They were arrangers. There was little information on the card, besides a picture of her and her ship, and a location she once frequented years ago. It was extremely impressive that her attackers had found her at all.
+
+There was another card. Same insignia. It was odd. Although arrangers would give the same card to as many assassins as would take them, they would rarely give one assassin more than one card at once. Leanne always assumed it because the cards themselves were expensive due to whatever cryptographic chips were on them, and that the arrangers didn't want the assassins hoarding the cards hoping they'd get lucky and kill one of the targets eventually.
 
-She checked the feeds from the cameras. It appeared she was alone, or alone enough. She rifled through the console. Her attackers, like her, were assassins. They should have a card for the job. It would have all the pertinent details, and they’d turn it in with proof of death. 
+On the second card was a woman. She lived on Dressara. Short black hair. Eyes aglow. Chin up. Her lips—outlined in dark red—halfway between a smile and a smirk. And, according to the card, not all that dangerous.
```

Comment:
I want her to have black hair in the rest of the story, so I'll change this to long black hair here.